packages/ds-sam-sdk/src/auction.ts

```diff
@@ -132,6 +131,46 @@ export class Auction {
     return winningTotalPmpe
   }
 
+  setStakeUnstakePriorities () {
+    this.data.validators.sort((a, b) => b.revShare.totalPmpe - a.revShare.totalPmpe)
+
+    this.data.validators.forEach((validator, index) => validator.stakePriority = index + 1)
```

Comment:
validators with same totalPmpe should have same stake prio

Reply:
Oh, then I misunderstood this part of the requirements, will update.